designer/server/src/main/scala/pl/touk/nussknacker/ui/validation/ScenarioPropertiesValidator.scala

```diff
@@ -60,7 +63,18 @@ class ScenarioPropertiesValidator(
 
     propertiesWithConfiguredValidator
       .collect { case (property, Some(config), validator: ParameterValidator) =>
-        validator.isValid(property._1, property._2, config.label).toValidatedNel
+        val expression = property._2
+        val value = if (expression.isBlank) {
```

Comment:
Nitpick: have you considered pattern matching here? like: 
```scala        
        val value = expression match {
          case ex if ex.isBlank => null
          case _ => Try(expression.toBoolean)
            .orElse(Try(expression.toInt))
            .orElse(Try(expression.toDouble))
            .getOrElse(expression)
        }
```

Reply:
done